shared/src/commonTest/kotlin/co/touchlab/kampstarter/BreedModelTest.kt

```diff
@@ -35,18 +35,18 @@ class BreedModelTest: BaseTest() {
     fun staleDataCheckTest() = runTest {
         settings.putLong(BreedModel.DB_TIMESTAMP_KEY, currentTimeMillis())
         assertFalse(ktorApi.jsonRequested)
-        model.getBreedsFromNetwork()
+        model.getBreedsFromNetwork().join()
         assertFalse(ktorApi.jsonRequested)
     }
 
     @Test
     fun updateFavoriteTest() = runTest {
-        model.getBreedsFromNetwork()
+        model.getBreedsFromNetwork().join()
         itemDataSummary.await(500)
```

Comment:
Do we still need the timeouts inside these awaits now that we have the joins?

Reply:
since the awaits waiting on a response from a separate flow, it could still take time for them to trigger and if a coding error means they never trigger we'd want it to fail rather than hang